recipes/microprofile/all/conanfile.py

```diff
@@ -0,0 +1,222 @@
+        "webserver_socket_buffer_size": 16 << 10,
+        "gpu_frame_delay": 5,
+        "name_max_length": 64,
+        "max_timers": 1024,
+        "max_threads": 32,
+        "max_string_length": 128,
+        "timeline_max_tokens": 64,
+        "thread_log_frames_reuse": 200,
+        "max_groups": 128,
+        "use_big_endian": False,
+        "enable_gpu_timer_callbacks": False,
+        "enable_gl_timers": False,
+        "enable_d3d11_timers": False,
+        "enable_d3d12_timers": False,
+        "enable_vulkan_timers": False
+    }
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+        else:
+            del self.options.enable_d3d11_timers
+            del self.options.enable_d3d12_timers
+
+    def _validate_max_groups(self):
+        try:
+            _max_groups = int(self.options.max_groups)
+            if _max_groups % 32 != 0:
+                raise ConanInvalidConfiguration("max_groups must be multiple of 32.")
+        except ValueError:
+            raise ConanInvalidConfiguration("max_groups must be a number.")
+
+    def _validate_timers(self):
+        _timers_enabled_count = 0
+        if self.options.enable_gl_timers:
+            _timers_enabled_count += 1
+        if self.options.get_safe("enable_d3d11_timers", self.default_options["enable_d3d11_timers"]):
+            _timers_enabled_count += 1
+        if self.options.get_safe("enable_d3d12_timers", self.default_options["enable_d3d12_timers"]):
+            _timers_enabled_count += 1
+        if self.options.enable_vulkan_timers:
+            _timers_enabled_count += 1
+        if _timers_enabled_count > 1:
+            raise ConanInvalidConfiguration("Cannot choose multiple GPU timers. Choose only one GPU timer.")
+        if _timers_enabled_count == 1 and self.options.enable_gpu_timer_callbacks:
+            raise ConanInvalidConfiguration("Cannot mix GPU callbacks and GPU timers.")
+
+    def validate(self):
+        if self.settings.compiler.get_safe("cppstd"):
+            tools.check_min_cppstd(self, 11)
+        self._validate_max_groups()
+        self._validate_timers()
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def requirements(self):
+        if self.options.use_miniz:
+            self.requires("miniz/2.2.0")
+        if self.options.enable_gl_timers:
+            self.requires("glad/0.1.34")
+        if self.options.enable_vulkan_timers:
+            self.requires("vulkan-loader/1.2.182")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version][0], strip_root=True, destination=self._source_subfolder)
+        tools.download(filename="LICENSE", **self.conan_data["sources"][self.version][1])
```

Comment:
```suggestion
        tools.download(filename="LICENSE", **self.conan_data["sources"][self.version][1])
```

Validate the sha256

Reply:
Doesn't the unpacking of the arguments takes care of that? Or am I missing something?

Reply:
You're right. The hash is handled correctly.